### PR TITLE
[Fix] Fix eplb bug and support fp8 load weight

### PR DESCRIPTION
**Pull request by xiaoxiaohehe001**
## Modifications

- 支持在 eplb 下 fp8 格式加载权重
- 修复 eplb 加载时传入 list 报错情况

## Checklist

- [x] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [x] Format your code, run `pre-commit` before commit.
- [x] Add unit tests. Please write the reason in this PR if no unit tests.
- [x] Provide accuracy results.
- [x] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

**Review by kevincheng2 — DISMISSED** [review]

File: fastdeploy/worker/worker_process.py [hunk 1]
```diff
@@ -260,13 +260,21 @@ def update_weights_from_tensor(self, mmap_infos):
...
+
+        while True:
+            if self.experts_manager.tensor_infos is None:
+                time.sleep(0.1)
+            else:
+                break
         state_dicts = load_tensor_from_shm_mem(self.experts_manager.tensor_infos, mmap_infos[MODEL_MAIN_NAME], logger)
         rank_expert_list, logical_to_physical_map, expert_count = self.experts_manager.get_ep_rank_to_expert_id_list()
-        self.worker.get_model().redundant_table_manger.update_expert_rank_table(
+        self.worker.get_model().ernie.redundant_table_manger.update_expert_rank_table(
```
Comment (yuanlehome) [inline fastdeploy/worker/worker_process.py]: 能把redundant_table_manger从ernie组网里移出来吗，这看起来eplb跟ernie强行绑定了，它应该是一个公共组件吧

**Review by yuanlehome — APPROVED** [review]
